pylimit/tests/test_pylimit.py

```diff
@@ -7,7 +7,8 @@
 class TestPyLimit(unittest.TestCase):
     def test_exception(self):
         limit = PyRateLimit()
```

Comment:
Thanks for the pull request. This pull request is also introducing a breaking change where constructor without arguments is not valid anymore. As a result, tests are not working anymore. Will it be possible to update constructor arguments to that tests don't fail?